### PR TITLE
Move monitor to scheduler

### PR DESCRIPTION
**Pull request by nzey**
We can do the same thing as the loop/sleep that was in `monitor.py` with the Heroku scheduler add-on.  The scheduler will just spin up the dyno to run every 10 minutes instead of constantly.

# TODO
- [ ] Add `python monitor.py` command to Heroku scheduler, https://cl.ly/d307e14049ac

### CODE REVIEW

File: monitor_job.py [hunk 1]
```diff
@@ -87,13 +86,11 @@ def record_restream_stats():
...
-        print "Monitor process sleeping for {}m".format(MINUTES_SLEEP)
-        time.sleep(SECONDS_SLEEP)
+    try:
+        record_all_metrics()
+        record_num_inputs()
+        record_restream_stats()
+    except Exception as e:
+        print e
+
+    print "Monitor process sleeping for {}m".format(MINUTES_SLEEP)
```
Comment (mrubens) [inline monitor_job.py]: Maybe update this copy?

Comment (mrubens) [reply monitor_job.py]: I think we can get rid of the sleep constants too

Comment (nzey) [reply monitor_job.py]: We are still using it in `record_metric` but I moved the environment variable lookup into that method.  

I then also noticed the we currently have the SLEEP_INTERVAL_MINUTES set to 1 (not the default 10 minutes).  I don't know the history  - maybe 10 minutes is ok.

Comment (mrubens) [reply monitor_job.py]: Oh hmm, I didn’t realize we made it shorter.

Comment (nzey) [reply monitor_job.py]: @thebrettd @joemanley201 Do you know why we have SLEEP_INTERVAL_MINUTES set to one and whether it's important to keep it at 1? If we use this Scheduler add-on, the smallest available interval is 10 minutes.

Comment (joemanley201) [reply monitor_job.py]: No, I don't know :(

**Comment by thebrettd** [comment]
I think we can close this

**Pull request closed without merging**